programs/server/Server.cpp

```diff
@@ -1344,8 +1344,8 @@ int Server::main(const std::vector<std::string> & /*args*/)
             settings.async_insert_max_data_size,
             AsynchronousInsertQueue::Timeout{.busy = settings.async_insert_busy_timeout_ms, .stale = settings.async_insert_stale_timeout_ms}));
 
-    /// Size of cache for marks (index of MergeTree family of tables). It is mandatory.
-    size_t mark_cache_size = config().getUInt64("mark_cache_size");
+    /// Size of cache for marks (index of MergeTree family of tables).
+    size_t mark_cache_size = config().getUInt64("mark_cache_size", 5368709120);
```

Comment:
Worth removing it from embedded config? https://github.com/ClickHouse/ClickHouse/blob/ae0d64d8e155ed8ce79f0845cbdc2de7b19de8da/programs/server/embedded.xml#L16

Reply:
Hm, good point. Guess the answer is yes :)

There is also a ton of config.xml templates used by tests ... will check if it's worth to change them too.